extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2064,3 +2058,23 @@ ANALYTICS_CONFIG['adobeanalytics_nativeConfig']
 
 ANALYTICS_CONFIG['oewa']['triggers']['pageview']['iframe' +
 /* TEMPORARY EXCEPTION */ 'Ping'] = true;
+
+/**
+ * @const {!JsonObject}
```

Comment:
Can you add documentation here on when vendors should add to this object including something about IAB viewability certified (assuming we already have such language elsewhere)?  Also, please add test coverage that ensures that vendor mappings are in the correct object (e.g. ANALYTICS_CONFIG does not have any transport: iframe entries and ANALYTICS_IFRAME_TRANSPORT_CONFIG only does).

Reply:
Doc: Done
Test coverage: Will do